cmd/kubelet/app/server_test.go

```diff
@@ -28,20 +28,20 @@ func TestValueOfAllocatableResources(t *testing.T) {
 		name           string
 	}{
 		{
-			kubeReserved:   map[string]string{"cpu": "200m", "memory": "-150G", "ephemeral-storage": "10Gi"},
+			kubeReserved:   map[string]string{"cpu": "200m", "memory": "-150G", "ephemeral-storage": "10Gi", "pid": "1000"},
 			systemReserved: map[string]string{"cpu": "200m", "memory": "15Ki"},
 			errorExpected:  true,
 			name:           "negative quantity value",
 		},
 		{
-			kubeReserved:   map[string]string{"cpu": "200m", "memory": "150Gi", "ephemeral-storage": "10Gi"},
+			kubeReserved:   map[string]string{"cpu": "200m", "memory": "150Gi", "ephemeral-storage": "10Gi", "pid": "0"},
 			systemReserved: map[string]string{"cpu": "200m", "memory": "15Ky"},
 			errorExpected:  true,
 			name:           "invalid quantity unit",
 		},
 		{
-			kubeReserved:   map[string]string{"cpu": "200m", "memory": "15G", "ephemeral-storage": "10Gi"},
-			systemReserved: map[string]string{"cpu": "200m", "memory": "15Ki"},
+			kubeReserved:   map[string]string{"cpu": "200m", "memory": "15G", "ephemeral-storage": "10Gi", "pid": "10000"},
+			systemReserved: map[string]string{"cpu": "200m", "memory": "15Ki", "swap": "100m"},
```

Comment:
Any reason we don't add these as separate test cases with names for more targeted testing instead of updating the existing ones?

Reply:
pid is missing in UT.
I will update swap in separated test cases.